src/microbe_stage/OrganelleDefinition.cs

```diff
@@ -317,6 +328,11 @@ public void Resolve(SimulationParameters parameters)
             LoadedScene = GD.Load<PackedScene>(DisplayScene);
         }
 
+        if (!string.IsNullOrEmpty(CorpseChunkScene))
```

Comment:
Please add a check on around line 292 that both CorpseChunkScene and DisplayScene can't be null or empty